index.html

```diff
@@ -392,7 +394,7 @@
   ctx.font = radius*0.20 + "px arial";
   ctx.textBaseline="middle";
   ctx.textAlign="center";
-  for(num = (max % incr === 0) ? max - incr : max - max % incr; num >= 0; num -= incr){

```

Comment:
I didn't have any special process. It just didn't look like anyone would deliberately remove that check, and there are also a lot of places where it looks like whitespace was added, even though Max didn't touch any code in those areas. So I went and found the commit where you made this change and cleaned up the whitespace.